src/test/java/com/sugarcrm/candybean/runner/BlockListUnitTest.java

```diff
@@ -50,8 +50,9 @@ public void ignoreFail() throws Exception {
 	public void blockListFail() throws Exception {
 		fail();
 	}
-	
+
 	// To ensure VRunner @Test doesn't break
+	@Ignore
```

Comment:
@Eriten This test class was giving "java.lang.Exception: No runnable methods" in production for no discernable reason -- it looked like a runnable method to me.  I recommended blocking it for now because we don't currently use VRunner or the blocklist feature, nor do we have plans to use them in the near future.  I have filed VOOD-276 to follow up and make a decision on whether there's any value in retaining some or part of VRunner.  Please let me know if that's a satisfactory resolution to this comment.